infra/infra.go

```diff
@@ -74,6 +76,58 @@ func main() {
 	k8sGKEResource.Command("delete", "gke resource delete -a service-account.json -f manifestsFileOrFolder -v PROJECT_ID:test -v ZONE:europe-west1-b -v CLUSTER_NAME:test -v hashStable:COMMIT1 -v hashTesting:COMMIT2").
 		Action(g.ResourceDelete)
 
+	// EKS based commands
+	e := eks.New()
+	k8sEKS := app.Command("eks", "Amazon Elastic Kubernetes Service - https://aws.amazon.com/eks").
+		Action(e.NewEKSClient)
+	k8sEKS.Flag("auth", "filename which consist eks credentials.").
+		PlaceHolder("credentials").
+		Short('a').
+		StringVar(&e.AuthFilename)
+	k8sEKS.Flag("file", "yaml file or folder  that describes the parameters for the object that will be deployed.").
+		Required().
+		Short('f').
+		ExistingFilesOrDirsVar(&e.DeploymentFiles)
+	k8sEKS.Flag("vars", "When provided it will substitute the token holders in the yaml file. Follows the standard golang template formating - {{ .hashStable }}.").
+		Short('v').
+		StringMapVar(&e.DeploymentVars)
+	k8sEKS.Flag("rangeVars", "Similar to vars but for range values which is stringified with seperator.").
```

Comment:
```suggestion
	k8sEKS.Flag("rangeVars", "Similar to vars, but for specifying range values.").
```

It'll also be super nice to have a `get_range`(usage) example in the helptext itself. :)